Index: icopy/makefile.win32
===================================================================
--- icopy/makefile.win32	(revision 8)
+++ icopy/makefile.win32	(working copy)
@@ -1,8 +1,8 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -ansi -O3 -pedantic -Wall
+CFLAGS = -c -ansi -O3 -pedantic -Wall
 LD = gcc
-LDFLAGS = -mno-cygwin 
+LDFLAGS = 
 
 CFILES = main.c
 HFILES = main.h
Index: makefile
===================================================================
--- makefile	(revision 8)
+++ makefile	(working copy)
@@ -1,44 +1,43 @@
 
-CC=gcc
-LD=gcc
+CC = gcc
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DHUC6280
+LD = gcc
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DHUC6280
 
-CFLAGS?= -c -O3 -ansi -pedantic -Wall
-LDFLAGS = -lm
-WLAFLAGS = $(CFLAGS) -DUNIX -DHUC6280
-
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
 OFILES = main.o parse.o include_file.o pass_1.o pass_2.o pass_3.o pass_4.o stack.o listfile.o
 
 
-all: $(OFILES) 
-	$(LD) $(LDFLAGS) $(OFILES) -o wla-huc6280 ; strip wla-huc6280
+all: $(OFILES) makefile
+	$(LD) $(LDFLAGS) $(OFILES) -o wla-huc6280.exe
 
-main.o: main.c defines.h main.h 
-	$(CC) $(WLAFLAGS) main.c
+main.o: main.c defines.h main.h makefile
+	$(CC) $(CFLAGS) main.c
 
-parse.o: parse.c defines.h parse.h 
-	$(CC) $(WLAFLAGS) parse.c
+parse.o: parse.c defines.h parse.h makefile
+	$(CC) $(CFLAGS) parse.c
 
-include_file.o: include_file.c defines.h include_file.h 
-	$(CC) $(WLAFLAGS) include_file.c
+include_file.o: include_file.c defines.h include_file.h makefile
+	$(CC) $(CFLAGS) include_file.c
 
-pass_1.o: pass_1.c defines.h pass_1.h parse.h  opcodes_gb.c opcodes_z80.c decode_gb.c
-	$(CC) $(WLAFLAGS) pass_1.c
+pass_1.o: pass_1.c defines.h pass_1.h parse.h makefile opcodes_gb.c opcodes_z80.c
+	$(CC) $(CFLAGS) pass_1.c
 
-pass_2.o: pass_2.c defines.h pass_2.h 
-	$(CC) $(WLAFLAGS) pass_2.c
+pass_2.o: pass_2.c defines.h pass_2.h makefile
+	$(CC) $(CFLAGS) pass_2.c
 
-pass_3.o: pass_3.c defines.h pass_3.h 
-	$(CC) $(WLAFLAGS) pass_3.c
+pass_3.o: pass_3.c defines.h pass_3.h makefile
+	$(CC) $(CFLAGS) pass_3.c
 
-pass_4.o: pass_4.c defines.h pass_4.h 
-	$(CC) $(WLAFLAGS) pass_4.c
+pass_4.o: pass_4.c defines.h pass_4.h makefile
+	$(CC) $(CFLAGS) pass_4.c
 
-stack.o: stack.c defines.h stack.h 
-	$(CC) $(WLAFLAGS) stack.c
+stack.o: stack.c defines.h stack.h makefile
+	$(CC) $(CFLAGS) stack.c
 
-listfile.o: listfile.c defines.h 
+listfile.o: listfile.c defines.h makefile
 	$(CC) $(WLAFLAGS) listfile.c
 
 
@@ -46,7 +45,7 @@
 
 
 clean:
-	rm -f $(OFILES) core *~ wla-huc6280
+	rm -f $(OFILES) *~ wla-huc6280.exe
 
 install:
-	cp wla-huc6280 /usr/local/bin
+	make ; cp wla-huc6280.exe /usr/local/bin
Index: makefiles/makefile.win32.6502
===================================================================
--- makefiles/makefile.win32.6502	(revision 8)
+++ makefiles/makefile.win32.6502	(working copy)
@@ -1,9 +1,9 @@
 
-CC?=gcc
-LD?=gcc
+CC=gcc
+LD=gcc
 
-CFLAGS?= -mno-cygwin -c -O3 -ansi -pedantic -Wall
-LDFLAGS = -mno-cygwin -lm -s
+CFLAGS= -c -O3 -ansi -pedantic -Wall
+LDFLAGS = -lm -s
 WLAFLAGS = $(CFLAGS) -DMSDOS -DMCS6502
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
Index: makefiles/makefile.win32.6510
===================================================================
--- makefiles/makefile.win32.6510	(revision 8)
+++ makefiles/makefile.win32.6510	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DMCS6510
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DMCS6510
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DMCS6510
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.65816
===================================================================
--- makefiles/makefile.win32.65816	(revision 8)
+++ makefiles/makefile.win32.65816	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DW65816
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DW65816
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DW65816
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.65c02
===================================================================
--- makefiles/makefile.win32.65c02	(revision 8)
+++ makefiles/makefile.win32.65c02	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DWDC65C02
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DWDC65C02
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DWDC65C02
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.gb
===================================================================
--- makefiles/makefile.win32.gb	(revision 8)
+++ makefiles/makefile.win32.gb	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DGB
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DGB
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DGB
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.huc6280
===================================================================
--- makefiles/makefile.win32.huc6280	(revision 8)
+++ makefiles/makefile.win32.huc6280	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DHUC6280
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DHUC6280
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DHUC6280
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.spc700
===================================================================
--- makefiles/makefile.win32.spc700	(revision 8)
+++ makefiles/makefile.win32.spc700	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DSPC700
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DSPC700
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DSPC700
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.z80
===================================================================
--- makefiles/makefile.win32.z80	(revision 8)
+++ makefiles/makefile.win32.z80	(working copy)
@@ -1,8 +1,9 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DZ80
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS -DWIN32 -DZ80
 LD = gcc
-LDFLAGS = -mno-cygwin -lkernel32
+LDFLAGS = -lkernel32
+WLAFLAGS = $(CFLAGS) -DMSDOS -DZ80
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
 HFILES = main.h parse.h include_file.h pass_1.h pass_2.h pass_3.h pass_4.h stack.h listfile.h defines.h
Index: makefiles/makefile.win32.z80.orig
===================================================================
--- makefiles/makefile.win32.z80.orig	(revision 8)
+++ makefiles/makefile.win32.z80.orig	(working copy)
@@ -2,8 +2,8 @@
 CC?=gcc
 LD?=gcc
 
-CFLAGS?= -mno-cygwin -c -O3 -ansi -pedantic -Wall
-LDFLAGS = -mno-cygwin -lm -s
+CFLAGS?= -c -O3 -ansi -pedantic -Wall
+LDFLAGS = -lm -s
 WLAFLAGS = $(CFLAGS) -DMSDOS -DZ80
 
 CFILES = main.c parse.c include_file.c pass_1.c pass_2.c pass_3.c pass_4.c stack.c listfile.c
Index: wlab/makefile
===================================================================
--- wlab/makefile	(revision 8)
+++ wlab/makefile	(working copy)
@@ -1,8 +1,7 @@
 
-CC?=egcs
-LD?=egcs
-
-CFLAGS?= -c -ansi -O3 -pedantic -Wall
+CC = gcc
+CFLAGS = -c -ansi -O3 -pedantic -Wall
+LD = gcc
 LDFLAGS = 
 
 CFILES = main.c
@@ -10,7 +9,7 @@
 OFILES = main.o
 
 all: $(OFILES) makefile
-	$(LD) $(LDFLAGS) $(OFILES) -o wlab ; strip wlab
+	$(LD) $(LDFLAGS) $(OFILES) -o wlab.exe
 
 main.o: main.c main.h makefile
 	$(CC) $(CFLAGS) main.c
@@ -20,7 +19,7 @@
 
 
 clean:
-	rm -f $(OFILES) core *~ wlab
+	rm -f $(OFILES) *~ wlab.exe
 
 install:
-	make ; cp wlab /usr/local/bin
+	make ; cp wlab.exe /usr/local/bin
Index: wlab/makefile.win32
===================================================================
--- wlab/makefile.win32	(revision 8)
+++ wlab/makefile.win32	(working copy)
@@ -1,8 +1,8 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -ansi -O3 -pedantic -Wall
+CFLAGS = -c -ansi -O3 -pedantic -Wall
 LD = gcc
-LDFLAGS = -mno-cygwin 
+LDFLAGS = 
 
 CFILES = main.c
 HFILES = main.h
Index: wlad/makefile
===================================================================
--- wlad/makefile	(revision 8)
+++ wlad/makefile	(working copy)
@@ -1,26 +1,25 @@
-
-CC?=egcs
-LD?=egcs
-
-CFLAGS?= -c -ansi -O3 -pedantic -Wall -DUNIX -DGB
-LDFLAGS = 
-
-CFILES = main.c
-HFILES = main.h
-OFILES = main.o
-
-all: $(OFILES) makefile
-	$(LD) $(LDFLAGS) $(OFILES) -o wlad ; strip wlad
-
-main.o: main.c main.h makefile
-	$(CC) $(CFLAGS) main.c
-
-
-$(OFILES): $(HFILES)
-
-
-clean:
-	rm -f $(OFILES) core *~ wlad
-
-install:
-	make ; cp wlad /usr/local/bin
+
+CC = gcc
+CFLAGS = -c -ansi -O3 -pedantic -Wall -DMSDOS -DGB
+LD = gcc
+LDFLAGS = 
+
+CFILES = main.c
+HFILES = main.h
+OFILES = main.o
+
+all: $(OFILES) makefile
+	$(LD) $(LDFLAGS) $(OFILES) -o wlad.exe
+
+main.o: main.c main.h makefile
+	$(CC) $(CFLAGS) main.c
+
+
+$(OFILES): $(HFILES)
+
+
+clean:
+	rm -f $(OFILES) *~ wlad.exe
+
+install:
+	make ; cp wlad.exe /usr/local/bin
Index: wlad/makefile.win32
===================================================================
--- wlad/makefile.win32	(revision 8)
+++ wlad/makefile.win32	(working copy)
@@ -1,8 +1,8 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -ansi -O3 -pedantic -Wall -DMSDOS -DGB
+CFLAGS = -c -ansi -O3 -pedantic -Wall -DMSDOS -DGB
 LD = gcc
-LDFLAGS = -mno-cygwin 
+LDFLAGS = 
 
 CFILES = main.c
 HFILES = main.h
Index: wlad_new/makefile.win32
===================================================================
--- wlad_new/makefile.win32	(revision 8)
+++ wlad_new/makefile.win32	(working copy)
@@ -1,8 +1,8 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -ansi -O3 -pedantic -Wall -DMSDOS -DGB
+CFLAGS = -c -ansi -O3 -pedantic -Wall -DMSDOS -DGB
 LD = gcc
-LDFLAGS = -mno-cygwin 
+LDFLAGS = 
 
 CFILES = main.c
 HFILES = main.h
Index: wlalink/makefile
===================================================================
--- wlalink/makefile	(revision 8)
+++ wlalink/makefile	(working copy)
@@ -1,17 +1,16 @@
 
-CC?=egcs
-LD?=egcs
+CC = gcc
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS
+LD = gcc
+LDFLAGS = 
 
-CFLAGS?= -c -O3 -ansi -pedantic -Wall
-LDFLAGS =
-
 CFILES = main.c memory.c parse.c files.c check.c analyze.c write.c compute.c discard.c listfile.c
 HFILES = main.h memory.h parse.h files.h check.h analyze.h write.h compute.h discard.h listfile.h defines.h
 OFILES = main.o memory.o parse.o files.o check.o analyze.o write.o compute.o discard.o listfile.o
 
 
 all: $(OFILES) makefile
-	$(LD) $(LDFLAGS) $(OFILES) -o wlalink ; strip wlalink
+	$(LD) $(LDFLAGS) $(OFILES) -o wlalink.exe
 
 
 main.o: main.c main.h defines.h
@@ -49,7 +48,7 @@
 
 
 clean:
-	rm -f $(OFILES) core *~ wlalink
+	rm -f $(OFILES) *~ wlalink.exe
 
 install:
-	make ; cp wlalink /usr/local/bin
+	make ; cp wlalink.exe /usr/local/bin
Index: wlalink/makefile.win32
===================================================================
--- wlalink/makefile.win32	(revision 8)
+++ wlalink/makefile.win32	(working copy)
@@ -1,8 +1,8 @@
 
 CC = gcc
-CFLAGS = -m486 -c -mno-cygwin -O3 -ansi -pedantic -Wall -DMSDOS
+CFLAGS = -c -O3 -ansi -pedantic -Wall -DMSDOS
 LD = gcc
-LDFLAGS = -mno-cygwin
+LDFLAGS = 
 
 CFILES = main.c memory.c parse.c files.c check.c analyze.c write.c compute.c discard.c listfile.c
 HFILES = main.h memory.h parse.h files.h check.h analyze.h write.h compute.h discard.h listfile.h defines.h
